systemtest/src/test/java/io/strimzi/systemtest/OpenShiftTemplatesTest.java

```diff
@@ -80,6 +80,36 @@ public void testStrimziPersistent() throws IOException {
         assertEquals("persistent-claim", mapper.readTree(cmData.get("zookeeper-storage")).get("type").asText());
     }
 
+    @Test
+    public void testStrimziPersistentWithCustomParameters() throws IOException {
+        Oc oc = (Oc) cluster.client();
```

Comment:
We should factor this out into a field of the OpenShiftTemplatesTest, since every method is going to want to use a downcast version of the client.

Reply:
Good catch!